docs/README.md

```diff
@@ -0,0 +1,70 @@
+# General guidelines for MDN translated content
+
+In this README you can find a collection of general guidelines for translating
+MDN content, which apply to every locale.
+
+For guidelines relating to specific locales, we have locale-specific docs in
+subdirectories:
+
+- [Russian translation guide / Участие в переводе](ru/translation-guide.md)
+
+If you want to add a guide to document some specific guidelines for your locale
+and it does not already appear here, you are welcome to add one, or
+[talk to the locale teams](https://github.com/mdn/translated-content/blob/docs-readme/README.md)
+about it. Similarly, if you can think of a good general guideline that you'd
+like to add here, feel free to create an issue to talk about it.
+
+## Translating heading IDs
+
+Our article headings are nearly always given IDs, so that we can automatically
+generate in-article navigation, identify code blocks to create live samples,
+and other reasons too. When translating headings, you should always translate
+the ID too, so for example:
+
+```html
+<h2 id="tutorials">Tutorials</h2>
+```
+
+in the fr locale would be
+
+```html
+<h2 id="tutoriels">Tutoriels</h2>
+```
+
+We generally advise that you write all IDs in lower-case. The platform
+automatically converts them at render time anyway, but keeping them lower-case
+means that there is less chance of a manually-created anchor link not working
+because of the conversion.
```

Comment:
Is there any technical reason to do that with Yari?
It seems to me that, with slugs not being translated anymore, it would a bit weird to have IDs translated (/foo/bar/baz#Baguette)

It doesn't look like IDs are dynamically generated as they were w/ Kuma.

Another benefit of keeping IDs untranslated is that it could allow comparison between en-US and localized page section per section.
For beginners' contribution, I would think it also makes it simpler not to bother with IDs.

Reply:
I mean, you are working on this content more than me, so if you'd rather not translate them, and that still works, then I'd be ok with that.